src/pinch-zoom.d.ts

```diff
@@ -0,0 +1,41 @@
+type PinchZoomEventHandler = (target: PinchZoom, event: TouchEvent) => void;
+
+interface IPinchZoomOptions {
```

Comment:
Why is this one named `IPinchZoomOptions` and not `PinchZoomOptions`?

Reply:
I think `I-prefix` refers to type `Interface`

Reply:
Alright! 😄 

Happy to hear our thoughts about my other questions above.

This is a nice addition, so would love to get it merged.

Reply:
I removed the comments, which should address those issues